src/inquirer/render/console/_checkbox.py

```diff
@@ -96,6 +96,12 @@ def process_input(self, pressed):
         elif pressed == key.RIGHT:
             if self.current not in self.selection:
                 self.selection.append(self.current)
+        elif pressed == key.CTRL_A:
+            self.selection = [i for i in range(len(self.question.choices))]
```

Comment:
should hitting `CTRL+A` a second time (if everything is selected), remove the selection?

Reply:
`[i for i in range(len(self.question.choices))]` feels unnecicaryliy complicated und non-pythonic, I need to think on this one (theres probably a smart, build in way to do this)

Reply:
I implemented it like LEFT or RIGHT. Both keys don't support toggle. `[i for i in range(len(self.question.choices))]` > this can be written as `[range(len(self.question.choices))]` Please suggest if there's any other optimal way.

Reply:
Yes! Adding some documentation.